source/DSCResources/DSC_CMForestDiscovery/DSC_CMForestDiscovery.schema.mof

```diff
@@ -0,0 +1,16 @@
+[ClassVersion("1.0.0"), FriendlyName("CMForestDiscovery")]
+class DSC_CMForestDiscovery: OMI_BaseResource
+{
+    [Key, Description("Specifies the SiteCode for the Configuration Manager site.")] String SiteCode;
+    [Required, Description("Specifies the enablement of the forest discovery method. If settings is set to $false no other value provided will be evaluated for compliance.")] Boolean Enabled;
+    [Write, EmbeddedInstance("DSC_CMForestDiscoveryPollingSchedule"), Description("Contains the polling schedule for Configuration Manager (RecurInterval, RecurCount)")] String PollingSchedule;
+    [Write, Description("Indicates whether Configuration Manager creates Active Directory boundaries from AD DS discovery information.")] Boolean EnableActiveDirectorySiteBoundaryCreation;
+    [Write, Description("Indicates whether Configuration Manager creates IP address range boundaries from AD DS discovery information.")] Boolean EnableSubnetBoundaryCreation;
+};
+
+[ClassVersion("1.0.0")]
+class DSC_CMForestDiscoveryPollingSchedule
+{
+     [Required, Description("Specifies the recur interval of days, hours, or minutes."),ValueMap{"Days","Hours","Minutes"},Values{"Days","Hours","Minutes"}] String RecurInterval;
```

Comment:
Minutes are not allowed in Forest Discovery

Reply:
Removed from schema and array (in the psm1)